__tests__/Car.test.js

```diff
@@ -0,0 +1,10 @@
+const { isInvalid } = require('../src/index');
+describe('Valid', () => {
+  test('Valid case1', () => {
+    expect(isInvalid('pobi,crong,honux')).toBe(false);
+  });
```

Comment:
`isInvalid` 처럼 부정문의 명칭보다는 긍정 구문으로 바꿔보면 어떨까요?
부정문은 한 번 더 **부정**에 대한 연산을 수행해야 하기 때문입니다.

```suggestion
  test('Valid case1', () => {
    expect(isValid('pobi,crong,honux')).toBe(true);
  });
```